.github/workflows/check-comment-reply.yml

```
@@ -0,0 +1,62 @@
+              | select(
+                  .in_reply_to_id == null
+                  and .user.login != $author
+                  and (.user.login as $u | $excluded | index($u) | not)
+              )
+            ]' all_comments.json > root_comments.json
+          ROOT_COUNT=$(jq length root_comments.json)
+          echo "🔢 Total Root Review Comments: $ROOT_COUNT"
+
+          echo "🔁 Filtering replies with @mention from non-excluded users..."
+          jq --argjson excluded "$(cat excluded_users.json)" '
+            [.[] 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)